README.md

```diff
@@ -1 +1,132 @@
-# buildroot-external-TI
\ No newline at end of file
+# TI Buildroot external tree
+
+Buildroot is a tool that simplifies and automates the process of building
+complete Linux systems for embedded devices. It simplifies the process
+of building a cross-compiled toolchain, a root filesystem, a Linux kernel,
+and a bootloader. This repository is a Buildroot `BR2_EXTERNAL` tree
+dedicated to supporting TI's SITARA platforms.
+
+The use of an external tree, such as buildroot-external-TI, allows for
+customization and extension of Buildroot without modifying the original
+Buildroot source. Upstream Buildroot also contains pre-defined
+configurations for SITARA platforms, but they use the upstream versions
+of U-Boot and Linux, while the configurations in this `BR2_EXTERNAL`
+tree uses the versions provided and supported by TI.
+
+## Available configurations
+
+This `BR2_EXTERNAL` tree provides two Buildroot configurations:
+
+1. `ti_release_am62x_sk_defconfig`, configuration to support the
+   [SK-AM62](https://www.ti.com/tool/SK-AM62) board. It builds and packages
+   TF-A, optee, ti-linux-firmware, ti-u-boot bootloader, ti-linux-kernel,
+   minimal user-space composed of Busybox and weston.
+
+2. `ti_release_am62x_sk_rt_defconfig`, configuration to support the
+   [SK-AM62](https://www.ti.com/tool/SK-AM62) board. It builds and packages
+   TF-A, optee, ti-linux-firmware, ti-u-boot bootloader, ti-linux-kernel (RT-Linux),
+   a minimal user-space composed of Busybox and weston.
+
```

Comment:
Add a section which says the prebuilt SD card image is already available to ti.com if users want to avoid building the entire image from sources.